composer.json

```diff
@@ -22,17 +22,23 @@
         "issues": "http://github.com/liip/RMT/issues"
     },
     "require": {
-        "php": ">=5.3.9",
+        "php": "^5.6|^7.0",
```

Comment:
It should be ^5.6|^7.1 according to your travis configuration or remove the build for 7.1 if not needed